### PR TITLE
Feat: Added landing page

### PR DESCRIPTION
**Pull request by ArjunGTX**
- Created Header
- Created Landing Page

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]
UI looking fantastic

File: index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>E-commerce Screens</title>
```
Comment (zabihshaik) [inline index.html:7]: Better to use Meta Tag for SEO

Comment (ArjunGTX) [reply index.html:7]: Since this is a demo page, didn't take those aspects into consideration. Thankyou for pointing it out

**Review by zabihshaik — APPROVED** [review]
Everything looks good to go.

File: index.html (hunk 1 above)
Comment (zabihshaik) [reply index.html:7]: Yeah. No Problem. :)

**Pull request merged**